pkg/model/volumes.go

```diff
@@ -25,6 +26,9 @@ import (
 
 const (
 	defaultVolumeSize = "5Gi"
+
+	// devPersistentVolumeEnabledEnvVar is the name of the environment variable to change the defaultVolumeSize value
+	devPersistentVolumeSizeEnvVar = "OKTETO_DEV_PERSISTENT_VOLUME_SIZE"
```

Comment:
Could you send a PR to document it [here](https://www.okteto.com/docs/reference/feature-flags/), please?